ferrariofilippo/woel
# [New Feature] Add `Edit Ad` page

Closes #22 

### Change list
- Replaced `create-ad-form` with a new `upsert-ad-form` used for both creation and editing
- Created `edit-ad` page
- Added a `trash` icon
- I changed a type in `supabase.ts` since that was causing an issue with `Textarea` component

### Notes
- Layout is not responsive yet
- There's a bug where images are not removed from `advertisement_picture` but that should be related to a db policy

### Test guide
1. Navigate to `localhost:3000`
2. Sign-in
3. Click on `Vendi subito`
4. Create an Ad
5. Check that everything is fine on the db
6. Navigate to `localhost:3000/edit?id=ADVERTISEMENT_ID`
7. Check that you can't edit ads belonging to others
8. Edit your ad
9. Check that everything is fine on the db
